src/main/webapp/populateData.js

```diff
@@ -89,3 +89,28 @@ function populateTasks() {
       });
 }
 
+/**
+ * Populate Calendar container with user's events
+ */
+function populateCalendar() {
+  const calendarContainer = document.querySelector('#calendar');
+  fetch('/calendar')
+      .then((response) => {
+        // If response is a 403, user is not authenticated
+        if (response.status === 403) {
+          throw new Error();
```

Comment:
Use the new AuthenticationError that I defined in pr #21 (it's new so you likely didn't see this yet). 

Then, you can actually check if the thrown error is an instance of AuthenticationError and only sign out in that case